cuztom.php

```diff
@@ -1,10 +1,9 @@
 <?php
 
+use Gizburdt\Cuztom\Cuztom;
```

Comment:
To my knowledge, this just makes Cuztom available in this file, and is mostly placed at the top of the file, after optional namespace